modules/core/src/test/scala/fs2/kafka/BaseKafkaSpec.scala

```diff
@@ -51,7 +56,10 @@ abstract class BaseKafkaSpec extends BaseAsyncSpec with EmbeddedKafka {
   final def withKafka[A](f: (EmbeddedKafkaConfig, String) => A): A =
     withRunningKafkaOnFoundPort(
       EmbeddedKafkaConfig(
-        customBrokerProperties = Map("transaction.state.log.replication.factor" -> "1")
+        customBrokerProperties = Map(
+          "transaction.state.log.replication.factor" -> "1",
+          "transaction.abort.timed.out.transaction.cleanup.interval.ms" -> transactionTimeoutInterval.toMillis.toString
```

Comment:
This is needed to force the broker to actually notice when open transactions time out.